scripts/release/assets/zip/index.html

```diff
@@ -27,39 +27,40 @@
 				}
 			}
 		</script>
-        <script type="module">
-            import {
-                ClassicEditor,
-                Essentials,
-                Paragraph,
-                Bold,
-                Italic,
-                Font
-            } from 'ckeditor5';
+		<script type="module">
+			import {
+				ClassicEditor,
+				Essentials,
+				Paragraph,
+				Bold,
+				Italic,
+				Font
+			} from 'ckeditor5';
 
-            ClassicEditor
-                .create( document.querySelector( '#editor' ), {
-                    plugins: [ Essentials, Paragraph, Bold, Italic, Font ],
-                    toolbar: [
+			ClassicEditor
+				.create( document.querySelector( '#editor' ), {
+					plugins: [ Essentials, Paragraph, Bold, Italic, Font ],
+					toolbar: [
 						'undo', 'redo', '|', 'bold', 'italic', '|',
 						'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
-					]
-                } )
-                .then( editor => {
-                    window.editor = editor;
-                } )
-                .catch( error => {
-                    console.error( error );
-                } );
-        </script>
-        <!-- A friendly reminder to run on a server, remove this during the integration. -->
-        <script>
-		        window.onload = function() {
-		            if ( window.location.protocol === "file:" ) {
-		                alert( "This sample requires an HTTP server. Please serve this file with a web server." );
-		            }
-		        };
+					],
+					licenseKey: 'GPL'
```

Comment:
```suggestion
					licenseKey: 'GPL' // Or <YOUR_LICENSE_KEY>.
```

Reply:
k, I'll update another PR with that.

Reply:
SHould we make it in this order?

Reply:
imho, it's much more user-friendly (if you ask about `GPL` and comment order)

Reply:
Sure, but we have it the other way round everywhere else.

Reply:
Agh, I'll fix that in the second PR.